src/v2/controllers/FormController.js

```diff
@@ -78,6 +78,7 @@ export default Controller.extend({
   createAfterEventContext () {
     const formName = this.options.appState.get('currentFormName');
     const authenticatorType = this.options.appState.get('authenticatorType');
```

Comment:
We still need `authenticatorType` here since we need it in `ViewClassNamesFactory`. We will address it in a separate PR.